mlflow/java/src/main/java/org/mlflow/sagemaker/ScoringServer.java

```diff
@@ -0,0 +1,270 @@
+package org.mlflow.sagemaker;
+
+import java.io.IOException;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.Optional;
+import org.mlflow.mleap.MLeapLoader;
+import org.mlflow.models.Model;
+import spark.Request;
+import spark.Response;
+import spark.Service;
+
+/** A RESTful webserver for {@link Predictor Predictors} that runs on the local host */
+public class ScoringServer {
+  public static final String RESPONSE_KEY_ERROR_MESSAGE = "Error";
+
+  public static final int HTTP_RESPONSE_CODE_SERVER_ERROR = 500;
```

Comment:
in jetty, `HttpServletResponse.SC_INTERNAL_ERROR`

Reply:
Done!